src/app/sponsors/sponsors.service.ts

```diff
@@ -21,10 +21,21 @@ export class SponsorsService {
     new Sponsor('collaboratory.png', new URL('https://www.cancercollaboratory.org/')),
     new Sponsor('oicr.png', new URL('https://oicr.on.ca/')),
     new Sponsor('ga4gh.png', new URL('https://genomicsandhealth.org/')),
-    new Sponsor('cwl.png', new URL('http://www.commonwl.org/'))
+    new Sponsor('ucsc.png', new URL('https://www.ucsc.edu/'))
```

Comment:
Hmmm, let's link to https://ucscgenomics.soe.ucsc.edu/ instead. 
(I know its technically a different logo, maybe add a comment to that effect rather than changing it now)